__tests__/app.test.js

```diff
@@ -3,34 +3,93 @@ const request = require("supertest");
+});
 
-describe("GET api/topics", () => {
-    describe("Connecting to path", () => {
-        test("Should return status 200", () => {
-            return request(app).get("/api/topics").expect(200)
+describe("GET /api/topics", () => {
+  describe("Connecting to path", () => {
+    test("Should return status 200", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("Should return the data from the path", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.length).toBe(data.topicData.length);
         });
-        test("Should return the data from the path", () => {
-            return request(app).get("/api/topics").expect(200)
-                .then(({ body }) => {
-                    expect(body.length).toBe(data.topicData.length)
-                });
+    });
+    test("Should return the correct data", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          body.forEach((topic) => {
+            expect(topic).toHaveProperty("slug", expect.any(String));
+            expect(topic).toHaveProperty("description", expect.any(String));
+          });
+        });
+    });
+  });
+});
+
+describe("GET /api", () => {
+  describe("Connecting to /api path", () => {
+    test("Should return status 200", () => {
+      return request(app).get("/api").expect(200);
+    });
+    test("Should return the data from the path", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body}) => {
+          expect(body).toMatchObject({...endpoints});
         });
-        test("Should return the correct data", () => {
-            return request(app).get("/api/topics").expect(200)
-                .then(({ body }) => {
-                    body.forEach((topic) => {
-                        expect(topic).toHaveProperty("slug", expect.any(String))
-                        expect(topic).toHaveProperty("description", expect.any(String))
-                    });
-                });
+    });
+    test("Should return the correct info for each endpoint (/api/topics)", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body["GET /api/topics"]).toMatchObject({
+            "description": "serves an array of all topics",
+            "queries": [],
+            "exampleResponse": {
+              "topics": [{ "slug": "football", "description": "Footie!" }]
+            }
+          });
         });
     });
-});
\ No newline at end of file
+    test("Should return the correct info for each endpoint (/api/articles)", () => {
+        return request(app)
+          .get("/api")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body["GET /api/articles"]).toMatchObject({
+                "description": "serves an array of all articles",
```

Comment:
Same thing here this test could be removed because the test on line _48_ has got you covered 👍